debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/xstream/LcrEventHandler.java

```diff
@@ -37,20 +41,28 @@
     private final RelationalDatabaseSchema schema;
     private final OracleOffsetContext offsetContext;
     private final boolean tablenameCaseInsensitive;
+    private final AtomicReference<PositionAndScn> lcrMessage;
```

Comment:
I'd suggest to not have the atomic reference here directly, but rather have a method `PositionAndScn getAndResetPublishedPosition()` on `XstreamStreamingChangeEventSource`.

Reply:
@gunnarmorling Done, I also added send message for symmetry